client/package.json

```diff
@@ -33,6 +33,6 @@
   },
   "lint-staged": {
     "*.{html,scss,css}": "prettier --check",
-    "*.{jsx,js}": "eslint --report-unused-disable-directives"
+    "*.{jsx,js}": "eslint --report-unused-disable-directives" 
```

Comment:
you should remove the unnecessary space at line 36

Reply:
true